### PR TITLE
Fix bug with display notification (instead of the OrderId is displaye…

### PR DESCRIPTION
**Pull request by voronkooleksandr**
…d the ID of the notification)

### CODE REVIEW

**Review by Mukola1983 — APPROVED** [review]
You should increase coverage))

**Review by hnativlyubomyr — APPROVED** [review]

File: src/app/ubs/ubs-user/ubs-user-messages/ubs-user-messages.component.html [hunk 1]
```diff
@@ -27,13 +27,13 @@
                     totalItems: count
                   }
           "
-          (click)="setRead(notification.id, notification.read)"
+          (click)="setRead(notification.orderId, notification.read)"
```
Comment (NewArtMask) [inline src/app/ubs/ubs-user/ubs-user-messages/ubs-user-messages.component.html:30]: Change please from **orderId** to **id**.

Comment (NewArtMask) [inline src/app/ubs/ubs-user/ubs-user-messages/ubs-user-messages.component.html:30]: Change please from **orderId** to **id**.

Comment (voronkooleksandr) [reply src/app/ubs/ubs-user/ubs-user-messages/ubs-user-messages.component.html]: Done! I hope it will be final stage our task :)

**Review by NewArtMask — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Review by melnychenkoas — APPROVED** [review]

**Review by krayevska — APPROVED** [review]

**Review by Mukola1983 — APPROVED** [review]

**Pull request merged**